en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

```diff
@@ -0,0 +1,346 @@
+| :---------- | :-------------- | :------------------------------------------------------------------------------------------ |
+| X           | SSHBACKEND      | Name of the backend: ```plink```                                                            |
+| X           | SSHUSERNAME     | By default, it uses the user running process ```centengine``` on your poller                |
+|             | SSHPASSWORD     | Can be used. If not set, SSH key authentication is used                                     |
+|             | SSHPORT         | By default: 22                                                                              |
+|             | SSHEXTRAOPTIONS | Customize it with your own if needed. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```          |
+
+**Warning** With that backend, you have to validate the target server fingerprint manually (with the SSHUSERNAME used).
+
+### libssh backend (by default)
+
+| Mandatory   | Name            | Description                                                                                 |
+| :---------- | :-------------- | :------------------------------------------------------------------------------------------ |
+| X           | SSHBACKEND      | Name of the backend: ```libssh```                                                           |
+| X           | SSHUSERNAME     | By default, it uses the user running process ```centengine``` on your poller                |
+|             | SSHPASSWORD     | Can be used. If not set, SSH key authentication is used                                     |
+|             | SSHPORT         | By default: 22                                                                              |
+|             | SSHEXTRAOPTIONS | Customize it with your own if needed. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```          |
+
+With that backend, you don't have to validate the target server fingerprint manually.
+
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test by running the following command (Parameters such as ```ssh-username``` or ```ssh-password```have to be adjusted):
+
+```bash
+ /usr/lib/centreon/plugins//centreon_linux_ssh.pl \
+    --plugin=os::linux::local::plugin \
+	--mode=cpu \
+	--hostname='myipaddress' \
+	--ssh-backend='libssh' \
+	--ssh-username='myuser' \
+	--ssh-password='mypassword' \
+	--ssh-port='22' \
+	--warning-core='60' \
+	--critical-core='70' \
+	--warning-average='60' \
+	--critical-average='75' \
+	--verbose \
+	--use-new-perfdata
+	
+	
+OK: CPU(s) average usage is 11.91 % - CPU '0' usage : 11.91 % |
+'cpu.utilization.percentage'=11.91%;;;0;100 '0#core.cpu.utilization.percentage'=11.91%;;;0;100 
+CPU '0' usage : 11.91 %
+```
+
+The command above gets the average of a Linux SSH CPU (```--mode=CPU```).
```

Comment:
Maybe a better explanation of the SSH options.

Reply:
Adding explanation in below lines